statistics/src/main/java/com/duckduckgo/app/statistics/api/PixelSender.kt

```diff
@@ -163,4 +187,25 @@ class RxPixelSender @Inject constructor(
     private fun getAtbInfo() = statisticsDataStore.atb?.formatWithVariant(variantManager.getVariantKey()) ?: ""
 
     private fun getDeviceFactor() = deviceInfo.formFactor().description
+
+    private suspend fun shouldFirePixel(
+        pixelName: String,
+        type: PixelType,
+    ): Boolean =
+        when (type) {
+            DEFAULT -> true
```

Comment:
I believe using `DEFAULT` doesn't provide much clarity for this type. Could we consider using terms like `ADHOC` or `ON_DEMAND` or something else that gives a more meaningful description?

Reply:
I don't feel strongly about this naming, in fact I also don't think `DEFAULT` is a particularly good name, but I was out of ideas. How about `COUNT` to match pixel naming convention that is already used in some modules? https://github.com/duckduckgo/Android/blob/dbae7fef6f47f31c7263e064469f2077675d308e/network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt#L29 ?

Reply:
Yes, COUNT makes sense.